postgres-appliance/configure_spilo.py

```diff
@@ -276,7 +276,7 @@ def get_placeholders(provider):
     placeholders.setdefault('PGDATA', os.path.join(placeholders['PGROOT'], 'pgdata'))
     placeholders.setdefault('PGUSER_STANDBY', 'standby')
     placeholders.setdefault('PGPASSWORD_STANDBY', 'standby')
-    placeholders.setdefault('USE_ADMIN', True)
+    placeholders.setdefault('USE_ADMIN', 'PGUSER_ADMIN' in placeholders)
```

Comment:
Actually I was talking about `PGPASSWORD_ADMIN`.

Admin user name already has sane default value and can be skipped from environment, but password you will probably override.

Reply:
Very true -- should I keep a defaulted `PGUSER_ADMIN` and not make it overridable?

Reply:
`placeholders.setdefault('PGUSER_ADMIN', 'admin )` -- this one just leave as it is.
If somebody will define a `PGUSER_ADMIN` environment variable with a different value, this value will override `admin`.

Reply:
Sorry about all the delays @CyberDem0n -- should be fixed now to check for the defined `PGPASSWORD_ADMIN` placeholder.